package/MDAnalysis/core/selection.py

```diff
@@ -805,17 +775,31 @@ def __init__(self, parser, tokens):
         uppers = []  # upper limit on any range
         lowers = []  # lower limit on any range
 
-        for val in values:
+        _values = []
+        while values:
+            v = values.pop(0)
+            if v in ("to", ":", "-"):
```

Comment:
Is "to" documented?

Reply:
Judging from the regex, `12to15` will also work.

Reply:
... yes, it's documented

Reply:
now "to" requires spaces